src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default class App extends React.Component {
```

Comment:
Not a biggie, but can we turn this class component to a functional component

Reply:
Done!

Reply:
Done!